.github/workflows/release-automation.yml

```diff
@@ -0,0 +1,24 @@
+name: Update flyteidl version 
+
+on:
+  schedule:
+    # TODO: Need to discuss
+    - cron:  '5 4 * * *'
+
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        REPOSITORY: [flyteadmin, flytepropeller, flyteconsole, flytekit]
```

Comment:
We do need to merge all the other PRs first before this, right?

Reply:
yes